deploy/freenas-iscsi-chap.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Secret
+apiVersion: v1
+type: kubernetes.io/iscsi-chap
+metadata:
+  namespace: kube-system
+  name: freenas-iscsi-chap
+stringData:
+  # Set the relevant auth credentials here
```

Comment:
This is new to me, let's make a note/comment here about which `base64` values are required or not.

Reply:
I would recommend the user to be familiar with iscsi pv settings, more info here: https://github.com/kubernetes/examples/tree/master/volumes/iscsi

Reply:
I see, wasn't familiar with stringData option generally. That's a gem. Let's throw that link somewhere in the project.

Reply:
Oh, you mean the stringData notion. :) For secrets, you can use stringData as a shorthand, that will be base64 encoded. :)

Reply:
Yeah, I had never seen that :)